accounting/filters/OsgScheddCpuRemovedFilter.py

```diff
@@ -0,0 +1,400 @@
+        # and sort them so we can easily grab the percentiles later
+        long_times_sorted = []
+        for (is_short, goodput_time, no_shadow, job_status) in zip(
+                is_short_job,
+                data["CommittedTime"],
+                data["_NoShadow"],
+                data["JobStatus"]):
+            if (is_short == False) and (no_shadow == False) and (job_status != 3):
+                long_times_sorted.append(goodput_time)
+        long_times_sorted = self.clean(long_times_sorted)
+        long_times_sorted.sort()
+
+        # Compute columns
+        row["All CPU Hours"]    = sum(self.clean(total_cpu_time)) / 3600
+        row["Good CPU Hours"]   = sum(self.clean(goodput_cpu_time)) / 3600
+        row["Num Uniq Job Ids"] = sum(data['_NumJobs'])
+        row["Num Rm'd Jobs"]    = sum([status == 3 for status in data["JobStatus"]])
+        row["Num Jobs Rm'd w/o exec"]= sum([starts == 0 for starts in data["NumJobStarts"]]) 
+        row["Avg MB Sent"]      = stats.mean(self.clean(data["BytesSent"], allow_empty_list=False)) / 1e6
+        row["Max MB Sent"]      = max(self.clean(data["BytesSent"], allow_empty_list=False)) / 1e6
+        row["Avg MB Recv"]      = stats.mean(self.clean(data["BytesRecvd"], allow_empty_list=False)) / 1e6
+        row["Max MB Recv"]      = max(self.clean(data["BytesRecvd"], allow_empty_list=False)) / 1e6
+        row["Num Short Jobs"]   = sum(self.clean(is_short_job))
+        row["Max Rqst Mem MB"]  = max(self.clean(data['RequestMemory'], allow_empty_list=False))
+        row["Med Used Mem MB"]  = stats.median(self.clean(data["MemoryUsage"], allow_empty_list=False))
+        row["Max Used Mem MB"]  = max(self.clean(data["MemoryUsage"], allow_empty_list=False))
+        row["Max Rqst Cpus"]    = max(self.clean(data["RequestCpus"], allow_empty_list=False))
+        row["Num Exec Atts"]    = sum(self.clean(num_exec_attempts))
+        row["Num Shadw Starts"] = sum(self.clean(num_shadow_starts))
+        row["Num Local Univ Jobs"] = sum(data["_NumLocalUnivJobs"])
+        row["Num Sched Univ Jobs"] = sum(data["_NumSchedulerUnivJobs"])
+
+        # Compute derivative columns
+        if row["All CPU Hours"] > 0:
+            row["% Good CPU Hours"] = 100 * row["Good CPU Hours"] / row["All CPU Hours"]
+        else:
+            row["% Good CPU Hours"] = 0
+        if row["Num Uniq Job Ids"] > 0:
+            row["Shadw Starts / Job Id"] = row["Num Shadw Starts"] / row["Num Uniq Job Ids"]
+        else:
+            row["Shadw Starts / Job Id"] = 0
+        if row["Num Shadw Starts"] > 0:
+            row["Exec Atts / Shadw Start"] = row["Num Exec Atts"] / row["Num Shadw Starts"]
+        else:
+            row["Exec Atts / Shadw Start"] = 0
+        if sum(data["_NumBadJobStarts"]) > 0:
+            row["CPU Hours / Bad Exec Att"] = (sum(badput_cpu_time) / 3600) / sum(data["_NumBadJobStarts"])
+        else:
+            row["CPU Hours / Bad Exec Att"] = 0
```

Comment:
Following suggested changes above
```suggestion
        if sum(data["_NumBadJobStarts"]) > 0:
            row["CPU Hours / Exec Att"] = (sum(badput_cpu_time) / 3600) / sum(data["_NumBadJobStarts"])
        else:
            row["CPU Hours / Exec Att"] = 0
```